.gitlab-ci.yml

```diff
@@ -1,8 +1,76 @@
 stages:
   - test
   - benchmark
+  - execute
   - upload
 
+
+# job_juwels_compute:
+#   stage: execute
+#   variables:
+#     SCHEDULER_PARAMETERS: '--account=cstma --nodes=1 --partition=devel'
+#   tags:
+#     - juwels
+#     - jacamar
+#     - compute
+#     - slurm
+#   artifacts:
+#     paths:
+#       - test.file
+#   script:
+#     - echo $SYSTEMNAME
+#     - touch test.file
+#   after_script:
+#     - hostname
+#     - id
+
+
+variables:
+  JUWELS_ACCOUNT: "cstma"
+
+
+test_JUWELS:
+  stage: benchmark
+  rules:
+    - if: $CI_COMMIT_MESSAGE !~ /.*\[CI-no-benchmarks\]/
+  tags:
+    - jacamar
+    - juwels
+    - login
+    - shell
+  artifacts:
+    when: always
+    paths:
+      - benchmarks
+      - sbatch.err
+      - sbatch.out
```

Comment:
Could we also have a directory where all job scripts can post their output? It would be neat to allow multiple job scripts. Not really needed, though.

Reply:
Yes sure we can have a directory for the outputs. Each job in Gitlab spawns in a separate directory. Therefore, multiple jobs (from Gitlab) won't put files in the same directory. If multiple tasks from slurm are spawned from a single Job in Gitlab, having this directory might make sense.